### DISCUSSION TITLE
Can not seem to access my paused instance from dashboard

### DISCUSSION THREAD

**Discussion by tyk3w** [supabase/supabase#20149] [Questions] [answered]
Hi,

I can not seem to access the instance from dashboard, I have already tried 3 different browsers and in every single one of them the instance just loads infinitely without seemingly accessing anything.

I noticed some API call in the console returns an error with code 500. (url: https://api.supabase.io/platform/projects/xxx)

Is that normal? My other instance/project, which is not paused, I can access as usual, so it must be some issue with my instance/project being paused or something.

![Screenshot of the infinite load issue](https://github.com/supabase/supabase/assets/79211356/918fb244-19f2-4a3a-9da3-907c190d1a18)

Thank you

**Comment by tyk3w** [comment]
Also this API call:

https://api.supabase.io/v1/projects/xxx/upgrade/eligibility 

returns code 544.

**Reply by felixyakubov91** [reply]
I have the same issue
Multiple devices, clearing cookies doesn't help. new project works

**Comment by Sebestala** [comment]
Same for me, got this error in the console:
GET https://api.supabase.io/v1/projects/xxxxxxxxxxxxxxxx/custom-hostname 400 (Bad Request)
GET https://api.supabase.io/platform/projects/xxxxxxxxxxxxxxxx 500 (Internal Server Error)

**Comment by Jongkeun** [comment]
Me either. It looks like the server has errors but the status is fine though.
https://status.supabase.com/

**Comment by Sebestala** [comment, marked as answer]
Seems to be fixed ! After a while my page proposed me to restore the project
